llvm/include/llvm/IR/IntrinsicsRISCV.td

```diff
@@ -177,7 +179,8 @@ let TargetPrefix = "riscv" in {
                     [LLVMMatchType<0>, llvm_ptr_ty,
                      LLVMScalarOrSameVectorWidth<0, llvm_i1_ty>,
                      llvm_anyint_ty, LLVMMatchType<1>],
-                    [NoCapture<ArgIndex<1>>, ImmArg<ArgIndex<4>>, IntrReadMem]>,
+                    [NoCapture<ArgIndex<1>>, ImmArg<ArgIndex<4>>, IntrReadMem,
```

Comment:
fault-only-first load may write vl, is this still `IntrArgMemOnly`?

Reply:
I presume no, from the definition of IntrArgMemOnly in Intrinsics.td:

> Other than reads from and (possibly volatile) writes to memory, it has no side effects.

Reply:
This class isn't used for fault-only-first loads. That's `RISCVUSLoadFF`. This patch isn't touching that intrinsic.

Reply:
Oh I misread it, I was reading the line below.